packages/shared-ux/prompt/no_data_views/src/no_data_views.stories.tsx

```diff
@@ -56,7 +58,22 @@ const componentArgTypes = {
 };
 
 export const NoDataViewsComponent = (params: Record<keyof typeof componentArgTypes, any>) => {
-  return <NoDataViewsPromptComponent onClickCreate={action('onClick')} {...params} />;
+  return <NoDataViewsPromptComponent {...params} />;
+};
+
+NoDataViewsComponent.play = async ({
```

Comment:
I'd love to see this included in the `mocks.ts` file, or at least exported for other stories to use.